cc/algorithms/algorithm.h

```diff
@@ -54,8 +54,11 @@ class Algorithm {
   //
```

Comment:
nit: Does this comment refer to epsilon in the constructor or epsilon as a class member? If it's the former, then I guess we should mention delta here too.

Reply:
Good point!  I think we should mention Definition 2.4 of this book (it is on page 17 and is the epsilon-delta definition).

Reply:
Fixed in the next commit